next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
```

Comment:
lol are you sure you want to do that ---^

Reply:
It wasn't me! Next regenerates this file.